src/EA.Weee.Requests/AatfEvidence/GetAatfNotesRequest.cs

```diff
@@ -21,16 +21,17 @@ public class GetAatfNotesRequest : EvidenceNoteFilterBaseRequest
 
         public DateTime? EndDateSubmitted { get; set; }
 
-        public GetAatfNotesRequest(Guid organisationId, 
-            Guid aatfId, 
-            List<NoteStatus> allowedStatuses,
-            string searchRef, Guid? recipient, WasteType? wasteType, NoteStatus? noteStatusFilter,
+        //public int ComplianceYear { get; set; }
+
+        public GetAatfNotesRequest(Guid organisationId, Guid aatfId, List<NoteStatus> allowedStatuses, string searchRef, int selectedComplianceYear, 
```

Comment:
rename selectedComplianceYear to complianceYear to allow json serialiser to match it